packages/trader/src/AppV2/Components/DealCancellation/deal-cancellation.tsx

```diff
@@ -0,0 +1,29 @@
+import { Localize } from '@deriv/translations';
+import React from 'react';
+import { getContractDetailsConfig } from 'AppV2/Utils/contract-details-config';
+import { observer } from '@deriv/stores';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import RiskManagementItem from '../RiskManagementItem';
+import { isValidToCancel, isOpen } from '@deriv/shared';
+import CardWrapper from '../CardWrapper';
+
+const DealCancellation = observer(() => {
+    const { contract_info } = useContractDetails();
+    const { is_deal_cancellation_visible } = getContractDetailsConfig(contract_info.contract_type ?? '');
+
+    const is_valid_to_cancel = isValidToCancel(contract_info);
+
+    return is_valid_to_cancel && is_deal_cancellation_visible && isOpen(contract_info) ? (
```

Comment:
Can we use `&&` here? Instead of ternary with `null`

Reply:
and maybe also camelCase? ;)

Reply:
No, if we use `&&` it would return false which is not a suitable value to return. Either return the component or null.